harness/determined/deploy/aws/cli.py

```diff
@@ -30,6 +30,28 @@ def validate(s: str) -> str:
     return validate
 
 
+def parse_custom_tags() -> Callable:
+    def parse(s: str) -> List[Tuple[str, str]]:
+        tags = []
+
+        for tag in s.split(","):
+            try:
+                key, value = tag.split("=", 2)
```

Comment:
I think this should be

```suggestion
                key, value = tag.split("=", 1)
```

because `a=b=c'.split('=', 2)` will produce 3 results, and this'll throw "too many values to unpack" etc.

Reply:
Ah yeah, you're right about that. This must be from when I was testing that out. Thanks for the catch.